roles/druid-common/templates/_common/common.runtime.properties

```diff
@@ -123,3 +128,5 @@ druid.storage.storageDirectory= {{ deep_storage_location }}
 #druid.processing.buffer.sizeBytes=1073741824
 druid.processing.buffer.sizeBytes=536870912
 
+druid.indexing.doubleStorage=double
```

Comment:
I thought I read this should not be enabled when doing a rolling upgrade from versions < 0.11?

Reply:
Correct. so in the rolling upgrade we do it afterwards as separate change. This merge request sets up a new cluster with 0.12 from the ground up so it can be set directly

Reply:
👍